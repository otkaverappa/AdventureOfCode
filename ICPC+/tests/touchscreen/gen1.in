19
s 10
c
g
a
x
t
s
y
n
k
p
b 10
n
b
o
y
r
h
m
w
z
g
d 10
i
e
o
x
y
h
q
z
b
s
u 10
l
r
q
a
b
y
s
t
g
f
w 10
u
o
p
e
a
m
c
i
r
k
p 10
b
y
n
p
z
d
q
t
a
e
l 10
q
k
m
x
c
h
d
j
f
a
x 10
a
g
z
b
k
j
c
v
y
i
e 10
n
z
e
y
p
x
t
v
k
l
f 10
z
i
y
g
n
u
m
p
t
j
v 10
s
a
i
y
u
d
c
e
j
z
f 10
b
l
v
n
e
s
z
p
a
y
z 10
j
h
m
y
v
c
x
e
z
s
y 10
h
y
u
m
a
b
w
s
g
l
d 10
d
b
j
e
a
z
w
y
x
f
f 10
m
l
a
q
d
g
r
f
y
z
b 10
b
n
d
c
j
w
k
x
z
o
x 10
j
m
x
b
i
y
a
p
f
v
d 10
i
l
q
o
c
v
b
d
y
a
