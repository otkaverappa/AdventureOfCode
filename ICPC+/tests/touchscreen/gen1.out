s 0
a 1
x 1
c 2
g 3
t 4
n 5
y 5
k 6
p 9
b 0
g 1
n 1
h 2
m 2
r 3
y 3
z 4
w 5
o 6
e 1
s 1
x 2
b 3
h 3
q 3
z 3
y 4
i 6
o 7
y 1
t 2
g 3
l 3
r 3
b 4
f 4
q 6
s 6
a 7
e 1
a 2
r 2
c 3
u 5
i 6
k 7
m 7
o 7
p 8
p 0
y 4
t 5
n 6
b 7
e 7
d 8
q 9
a 10
z 11
k 1
j 2
h 3
m 3
f 5
d 6
c 7
a 8
x 8
q 9
c 1
z 1
a 2
v 2
b 3
g 4
j 6
y 6
k 7
i 8
e 0
t 2
v 3
x 3
y 3
z 4
n 5
k 6
l 7
p 7
g 1
t 2
j 3
n 3
y 3
m 4
u 4
z 4
i 5
p 7
c 1
d 2
e 3
s 3
z 3
a 4
j 4
y 4
u 5
i 6
v 1
b 2
e 2
s 2
a 3
n 3
y 3
z 4
l 5
p 7
z 0
x 1
c 2
s 2
v 3
e 4
h 6
m 6
j 7
y 7
y 0
h 1
u 1
g 2
b 3
m 3
l 4
w 4
s 5
a 6
d 0
e 1
f 1
a 2
w 2
x 2
b 3
z 3
j 4
y 4
f 0
d 1
g 1
r 1
a 3
y 3
m 4
q 4
z 4
l 5
b 0
n 1
c 2
d 3
j 3
x 3
k 4
z 4
w 5
o 6
x 0
a 2
v 2
b 3
f 3
m 5
j 6
y 6
i 8
p 10
d 0
c 1
a 2
v 2
b 3
q 3
y 4
i 6
l 6
o 7
